include/kyosu/functions/zeta.hpp

```diff
@@ -56,8 +56,7 @@ namespace kyosu
 //!   @code
 //!   namespace kyosu
 //!   {
-//!      template<eve::floating_ordered_value T> constexpr auto zeta(T z) noexcept;
-//!      template<kyosu::concepts::complex T>  constexpr auto zeta(T z) noexcept;
+//!      constexpr auto zeta(auto z) noexcept;
```

Comment:
This is false, I don't think we can call zeta with a string ?

Reply:
done